src/include/daos_mgmt.h

```diff
@@ -33,108 +33,6 @@ typedef struct {
 	d_rank_list_t			*mgpi_svc;
 } daos_mgmt_pool_info_t;
 
-/**
- * Currently, the following methods are mostly for dmg and tests.
- */
-
-/**
- * Exclude a set of storage targets from a pool.
- *
- * \param uuid	[IN]	UUID of the pool
- * \param grp	[IN]	process set name of the DAOS servers managing the pool
- * \param tgts	[IN]	Target to be excluded from the pool.
- *			Now can-only exclude one target per API calling. If
- *			tl_tgts = -1, it means it will exclude all targets
- *			on the rank.
- * \param ev	[IN]	Completion event, it is optional and can be NULL.
- *			The function will run in blocking mode if \a ev is NULL.
- *
- * \return		These values will be returned by \a ev::ev_error in
- *			non-blocking mode:
- *			0		Success
- *			-DER_INVAL	Invalid parameter
- *			-DER_UNREACH	Network is unreachable
- *			-DER_NONEXIST	Pool is nonexistent
- */
-int
-daos_pool_tgt_exclude(const uuid_t uuid, const char *grp,
```

Comment:
is this function still used in the vol-daos repository, in the h5daos_test_recovery test? Linking to the vol-daos tag 1.1.0rc2 that I believe the daos-stack/hdf5-vol-daos packaging repository refers to as the vol-daos code to use.
 
https://github.com/HDFGroup/vol-daos/blob/v1.1.0rc2/test/daos_vol/h5daos_test_recovery.c#L136

Reply:
Good finding, thanks. Sent a change proposal to Ray Lu for h5daos_test_recovery.c.
At the mean while, we should think whether we can go ahead land this PR, or need to wait for changes in h5daos_test_recovery.c landed?

Reply:
Well, this would break the HDF5 test ... so we need to do this in the proper order.
BTW, i think that some associated python functions should be removed too. @phender could you please comment?

Reply:
Does/can some kind of shared library major version need to be incremented to indicate the breakage?

Otherwise there is no "proper order".  At any given time there can be PRs in flight on either side of an API breakage.  The only nice/fair way to handle this is to provide compatibility to both sides.  The way this is done with shared libraries is to increment the SONAME (major version digit) of the library.

For non-library API, one typically has to provide both new and old APIs for at least some period of time.

Is `src/include/daos_mgmt.h` a client (libdaos) header file?  If so, this looks like an ABI breaking change in the client and as such, it should increment the libdaos SONAME.